src/components/manage_users_table/ManageUsersTableCellData.js

```diff
@@ -17,51 +18,44 @@ export function usernameCellData({displayName, userId, label= 'user-name'}) {
   };
 }
 
-export function googleIdCellData({userId, email, label = 'google-id'}) {
+export function emailCellData({userId, email, label = 'email'}) {
   return {
     data: email,
     value: email,
     id: userId,
     style: {
-      color: styles.color.googleId,
-      fontSize: styles.fontSize.googleId,
+      color: styles.color.email,
+      fontSize: styles.fontSize.email,
       fontWeight: '500'
     },
     label
   };
 }
 
-//Redirect for admin review page, only used in admin manage dar collections table
-export function roleCellData({userId, roles, label = 'user-role'}) {
+export function permissionsCellData({userId, roles, libraryCards, institution, label = 'permissions'}) {
+  const hasLibraryCard = libraryCards?.map((lc) => lc.institutionId)?.includes(institution?.id);
+  const roleNames = map(roles, 'name').filter((roleName) => roleName !== 'Researcher');
+  const perms = (hasLibraryCard ? roleNames.concat('Library Card') : roleNames);
```

Comment:
The current search bar filters the table on `displayName`, `email`, and `roles`. Since `Library Card` is a new addition to the permissions column, you'll need to update the search function to account for that as well.